build/antrea/base/Dockerfile.ubi

```diff
@@ -0,0 +1,22 @@
+FROM ubuntu:20.04 as cni-binaries
+
+RUN apt-get update && \
+    apt-get install -y --no-install-recommends wget ca-certificates
+
+# Leading dot is required for the tar command below
+ENV CNI_PLUGINS="./host-local ./loopback ./portmap ./bandwidth"
+
+RUN mkdir -p /opt/cni/bin && \
+    wget -q -O - https://github.com/containernetworking/plugins/releases/download/v0.8.7/cni-plugins-linux-amd64-v0.8.7.tgz | tar xz -C /opt/cni/bin $CNI_PLUGINS
+
+
+FROM antrea/ovs-ubi:2.14.0
+
+LABEL maintainer="Antrea <projectantrea-dev@googlegroups.com>"
+LABEL description="Takes care of building the Antrea binaries as part of building the image."
+
+USER root
+
+RUN yum install ipset jq -y
+
+COPY --from=cni-binaries /opt/cni/bin /opt/cni/bin
```

Comment:
Can we put this image together with the final antrea image?
cni-binaries + ovs-ubi:2.14.0 + antrea-build => final image

Reply:
Add no need to have a base Dockerfile, we can put it in the same file `build/antrea/Dockerfile.ubi`?

Reply:
> Add no need to have a base Dockerfile, we can put it in the same file `build/antrea/Dockerfile.ubi`?

Currently the structure of image and Dockerfile totally follows the structure in Antrea repo. Maybe we could consider putting it in the same Dockerfile.

Reply:
> Can we put this image together with the final antrea image?
> cni-binaries + ovs-ubi:2.14.0 + antrea-build => final image

We only need to copy part of building results to the final image, and maybe we shouldn't put the whole image into the final image?